src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java

```diff
@@ -203,7 +203,7 @@ public void testGetLastRevisionsQueryAfterDeletion() throws IOException, Interru
     Assert.assertTrue(storage.createRevisionEntry(getDummyGitData("5", time5)));
 
     Assert.assertTrue(storage.deleteRevisionEntry("3"));
-  
+
```

Comment:
This change doesn't belong in the scope of this PR

Reply:
```suggestion
  
```